src/components/NewAppModal.js

```diff
@@ -57,7 +57,9 @@ export const NewAppModal = _.flow(
         return h(ButtonPrimary, { onClick: () => createApp() }, ['Create'])
       }],
       ['launchWarn', () => {
-        return h(GalaxyLaunchButton, { app, onClick: onDismiss })
+        return h(GalaxyLaunchButton, {
+          app, onClick: () => onDismiss()
+        })
```

Comment:
nit: This looks left-over from the previous implementation, it would be nice to remove this diff.